shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -258,7 +258,7 @@ class DiffTests
           case "dv" => mode.copy(debugVariance = true)
           case "ge" => mode.copy(expectCodeGenErrors = true)
           case "re" => mode.copy(expectRuntimeErrors = true)
-          case "r" | "showRepl" => mode.copy(showRepl = true)
+          case "ShowRepl" => mode.copy(showRepl = true)
```

Comment:
Why would you do that? There's a reason I changed it to lowercase... Only commands that modify the global file state are upper case!

Reply:
This is not a deliberate change. It was accidentally caused when I switched branches.